app/views/supervisor/courses/show.html.erb

```diff
@@ -1,20 +1,105 @@
+              <th>Name</th>
+              <th>Created by</th>
+              <th>Status</th>
+              <th>Start date</th>
+              <th>Finish date</th>
+              <th></th>
+            </tr>
+            <% @course_subjects.each do |course_subject| %>
+              <tr>
+                <td><%= course_subject.subject.title %></td>
+                <td><%= course_subject.subject.creator.name %></td>
+                <td>
+                  <%= link_to supervisor_update_status_path(@course.id, course_subject.subject_id),
+                    method: :patch,
+                    remote: true,
+                    class: "btn btn-sm #{course_subject.status? ? "btn-success" : "btn-danger"}" do %>
+                    <i class="fas <%= course_subject.status? ? "fa-toggle-on" : "fa-toggle-off"%>"></i>
+                  <% end %>
+                </td>
+                <td><%= course_subject.start_date %></td>
+                <td><%= course_subject.finish_date %></td>
+                <td>
+                  <%= link_to supervisor_edit_course_subjects_path(@course.id, course_subject.subject_id),
+                          class: "btn btn-warning btn-sm" do %>
+                    <i class="fas fa-edit"></i>
+                  <% end %>
+                  <%= link_to supervisor_remove_subjects_path(@course.id, course_subject.subject_id),
+                          method: :delete,
+                          remote: true,
+                          data: {confirm: "Are you sure?"},
+                          class: "btn btn-danger btn-sm" do %>
+                    <i class="fas fa-minus"></i>
+                  <% end %>
+                </td>
+              </tr>
+            <% end %>
+          </table>
+        </div>
+      </div>
+    </div>
   </div>
-  <div class="card-body">
-    <p>
-      <strong><%= t(:description) %>:</strong>
-      <%= @course.description %>
-    </p>
-    <p>
-      <strong><%= t(:created_by) %>:</strong>
-      <%= @course.creator.name %>
-    </p>
-    <hr>
-    <p>
-      <strong><%= t(:subject_list) %></strong>
-    </p>
+  <div class="col-6">
+    <div class="card shadow mb-4">
+      <!-- Card Header - Accordion -->
+      <a href="#available-subjects" class="d-block card-header py-3" data-toggle="collapse" role="button" aria-expanded="true" aria-controls="collapseCardExample">
+        <h6 class="m-0 font-weight-bold text-primary">Available subjects</h6>
+      </a>
+      <!-- Card Content - Collapse -->
+      <div class="collapse show" id="available-subjects">
+        <div class="card-body">
+          <table class="table table-bordered table-hover">
+            <tr>
+              <th>Name</th>
+              <th>Created by</th>
+              <th>Create date</th>
+              <th></th>
+            </tr>
+            <% @subjects.each do |subject| %>
+              <tr>
+                <td><%= subject.title %></td>
+                <td><%= subject.creator.name %></td>
```

Comment:
`subject.creator.name`
tìm hiểu về `delegate in rails `hì

Reply:
fixed